src/main/java/org/example/algorithms/KaddensAlgo.java

```diff
@@ -0,0 +1,17 @@
+public class KaddensAlgo {
```

Comment:
Please add package info at the top of the class.
```java
package org.example.algorithms;
```

See other classes for reference. Thanks.